deno_typescript/ops.rs

```diff
@@ -0,0 +1,160 @@
+use crate::get_asset;
+use crate::TSState;
+use deno::CoreOp;
+use deno::ErrBox;
+use deno::ModuleSpecifier;
+use deno::Op;
+use deno::OpId;
+use serde::Deserialize;
+use serde_json::json;
+use serde_json::Value;
+
+#[derive(Debug)]
+pub struct WrittenFile {
+  pub url: String,
+  pub module_name: String,
+  pub source_code: String,
+}
+
+fn dispatch2(
+  s: &mut TSState,
+  op_id: OpId,
+  control_buf: &[u8],
+) -> Result<Value, ErrBox> {
+  let v = serde_json::from_slice(control_buf)?;
+  match op_id {
+    49 => read_file(s, v),
```

Comment:
Add a comment that these need to be kept in sync with the other places OP IDs are hard coded.
(Maybe add a mod op_ids so they are hard coded only once in rust?)

Reply:
> Add a comment that these need to be kept in sync with the other places OP IDs are hard coded.

Done.

> (Maybe add a mod op_ids so they are hard coded only once in rust?)

I think we need a better solution that doesn't have any runtime overhead, which can also be used in the CLI. I like the `derive(TypeScript)` idea... but yea. That's for a different PR.